src/Nancy/Responses/TextResponse.cs

```diff
@@ -10,29 +10,36 @@
     /// </summary>
     public class TextResponse : Response
     {
+        private const string TextPlainContentType = "text/plain";
+
         /// <summary>
         /// Creates a new instance of the TextResponse class
         /// </summary>
         /// <param name="contents">Text content - defaults to empty if null</param>
         /// <param name="contentType">Content Type - defaults to text/plain</param>
         /// <param name="encoding">String encoding - UTF8 if null</param>
-        public TextResponse(string contents, string contentType = "text/plain", Encoding encoding = null)
+        public TextResponse(string contents, string contentType = null, Encoding encoding = null)
         {
             if (encoding == null)
             {
                 encoding = Encoding.UTF8;

```

Comment:
I was thinking that perhaps we should add an `Encoding` settings to the new config API (probably along with stuff from static configuration, such as case sensitivity) that it would use unless an encoding was explicitly declared. This pull-request can still go in, but we should add it to the task list in #2000